uarray/tests/test_uarray.py

```diff
@@ -16,20 +16,23 @@ class Backend:
     __ua_domain__ = "ua_tests"
 
 
+def create_nullary_mm(default=None):
```

Comment:
Can you convert this to a PyTest fixture? That’s the standard way to handle things like these,

Reply:
Done